Rename PruningRules#includes to pruningRoots

oldergod opened the pull request.

jrodbx commented:
WireRun has a concept of treeShakingRoots, making this rename a bit confusing.  What makes `includes` no longer an adequate descriptive name?

oldergod commented:
PruningRules is used both as pruning rules and as emitting rules. I wanna
split that into two different classes. For the pruning one, include means
root so I renamed accordingly.

WireRun will be using the new EmittingRules in follow-up

On Thu., Jan. 2, 2020, 16:58 John Rodriguez, <notifications@github.com>
wrote:

> WireRun has a concept of treeShakingRoots, making this rename a bit
> confusing. What makes includes no longer an adequate descriptive name?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/square/wire/pull/1351?email_source=notifications&email_token=AANPR5OI4JCI4FQUMSM4SWLQ3ZPR5A5CNFSM4KCHCRXKYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEH7R7XQ#issuecomment-570367966>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AANPR5ILY72EDIV6WR4WGXDQ3ZPR5ANCNFSM4KCHCRXA>
> .
>
